charts/stable/nextcloud/values.yaml

```diff
@@ -0,0 +1,438 @@
+              httpHeaders:
+                Host: kube.internal.healthcheck
+            startup:
+              enabled: true
+              type: tcp
+              port: "{{ .Values.service.main.ports.main.port }}"
+  notify:
+    enabled: true
+    type: Deployment
+    strategy: RollingUpdate
+    replicas: 1
+    podSpec:
+      containers:
+        notify:
+          primary: true
+          enabled: true
+          imageSelector: hpbImage
+          envFrom:
+            - configMapRef:
+                name: hpb-config
+          probes:
+            readiness:
+              enabled: true
+              path: /push/test/cookie
+              port: 7867
+              httpHeaders:
+                Host: kube.internal.healthcheck
+            liveness:
+              enabled: true
+              path: /push/test/cookie
+              port: 7867
+              httpHeaders:
+                Host: kube.internal.healthcheck
+            startup:
+              enabled: true
+              type: tcp
+              port: 7867
+  imaginary:
+    enabled: true
+    type: Deployment
+    strategy: RollingUpdate
+    replicas: 1
+    podSpec:
+      containers:
+        imaginary:
+          primary: true
+          enabled: true
+          imageSelector: imaginaryImage
+          command: imaginary
+          args:
+            - -p
+            - "{{ .Values.service.imaginary.ports.imaginary.port }}"
+            - -concurrency
+            - "10"
+            - -enable-url-source
+            - -return-size
+          probes:
+            readiness:
+              enabled: true
+              path: /health
+              port: "{{ .Values.service.imaginary.ports.imaginary.port }}"
+            liveness:
+              enabled: true
+              path: /health
+              port: "{{ .Values.service.imaginary.ports.imaginary.port }}"
+            startup:
+              enabled: true
+              type: tcp
+              port: "{{ .Values.service.imaginary.ports.imaginary.port }}"
+  clamav:
+    enabled: true
+    type: Deployment
+    strategy: RollingUpdate
+    replicas: 1
+    podSpec:
+      containers:
+        clamav:
+          primary: true
+          enabled: true
+          imageSelector: clamavImage
+          # FIXME: https://github.com/Cisco-Talos/clamav/issues/478
+          securityContext:
+            runAsUser: 0
+            runAsGroup: 0
+            runAsNonRoot: false
+            readOnlyRootFilesystem: false
+          envFrom:
+            - configMapRef:
+                name: clamav-config
+          probes:
+            readiness:
+              enabled: true
+              type: exec
+              command: clamdcheck.sh
+            liveness:
+              enabled: true
+              type: exec
+              command: clamdcheck.sh
+            startup:
+              enabled: true
+              type: exec
+              command: clamdcheck.sh
```

Comment:
```suggestion
            startup:
              enabled: true
              type: tcp
              port: "{{ .Values.service.clamav.ports.clamav.targetPort }}"
```

Reply:
Not 200% sure tcp is optimal here, but it's two staged anyway...